vars/folioParameters.groovy

```diff
@@ -35,67 +35,71 @@ private def _paramExtendedSingleSelect(String name, String reference, String scr
 }
 
 def agent() {
-    return _paramChoice('AGENT', Constants.JENKINS_AGENTS, 'Select build agent')
+    return _paramChoice('AGENT', Constants.JENKINS_AGENTS, 'Select Jenkins agent for build')
 }
 
 def refreshParameters() {
-    return _paramBoolean('REFRESH_PARAMETERS', false, 'Set to true for job parameters refresh')
+    return _paramBoolean('REFRESH_PARAMETERS', false, 'Set to true for update pipeline parameters, it will not run a pipeline')
 }
 
 def cluster() {
-    return _paramChoice('CLUSTER', Constants.AWS_EKS_CLUSTERS, '(Required) Select cluster')
+    return _paramChoice('CLUSTER', Constants.AWS_EKS_CLUSTERS, '(Required) Select cluster for current job')
 }
 
 def namespace() {
-    return _paramExtendedSingleSelect('NAMESPACE', 'CLUSTER', folioStringScripts.getNamespaces(), '(Required) Select cluster namespace')
+    return _paramExtendedSingleSelect('NAMESPACE', 'CLUSTER', folioStringScripts.getNamespaces(), '(Required) Select cluster namespace for current job')
 }
 
 def repository() {
-    return _paramChoice('FOLIO_REPOSITORY', repositoriesList(), 'Select source repository')
+    return _paramChoice('FOLIO_REPOSITORY', repositoriesList(), 'Platform-complete repository, complete stripes platform consists an NPM package.json that specifies the version of @folio/stripes-core')
 }
 
 def branch(String paramName = 'FOLIO_BRANCH', String repository = 'platform-complete') {
     return _paramExtendedSingleSelect(paramName, '', folioStringScripts.getRepositoryBranches(repository), "(Required) Select what '${repository}' branch use for build")
 }
 
 def okapiVersion() {
-    return _paramExtendedSingleSelect('OKAPI_VERSION', 'FOLIO_BRANCH', folioStringScripts.getOkapiVersions(), 'Select okapi version')
+    return _paramExtendedSingleSelect('OKAPI_VERSION', 'FOLIO_BRANCH', folioStringScripts.getOkapiVersions(), 'Selectwhat Okapi version use for build')
 }
 
 def loadReference(boolean value = true) {
-    return _paramBoolean('LOAD_REFERENCE', value, 'Set to true to load reference data during install')
+    return _paramBoolean('LOAD_REFERENCE', value, 'Select true to load reference data during install')
```

Comment:
Please add more description
Select true to load reference data during installation; this will cause the modules to load all reference data.